GFS_layer/GFS_typedefs.F90

```diff
@@ -2761,6 +2831,15 @@ subroutine control_initialize (Model, nlunit, fn_nml, me, master, &
       endif
     endif
 
+    ! -- CHECK for ntke if using satmedmf
+    if (Model%satmedmf) then
+       if (Model%ntke < 1 .or. Model%ntke > Model%ntrac) then
+          write(*,*) ' FATAL GFS_typedefs: TKE PBL scheme enabled (satmedmf) but TKE tracer not found in field_table.'
+          write(*,*) ' Stopping execution.'
+          stop 999
```

Comment:
Stop statements may not abort a program correctly when using MPI.  Should either import and use mpp_error or make a call to MPI_Finalize to properly abort execution.

Reply:
A few other places in this subroutine use the stop function to terminate the program. Should all of them be revised?